vcd/resource_vcd_vapp_vm.go

```diff
@@ -1614,7 +1614,7 @@ func resourceVcdVAppVmDelete(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	log.Printf("[TRACE] Removing VM: %s", vm.VM.Name)
-
+	// sleep() sad
```

Comment:
Leftover from the other issue?

Reply:
Oops. Total leftover from troubleshooting another issue.

Reply:
Removed